schema/manifest_backwards_compatibility_test.go

```diff
@@ -190,36 +190,36 @@ func TestBackwardsCompatibilityManifest(t *testing.T) {
 
 func TestBackwardsCompatibilityConfig(t *testing.T) {
 	for i, tt := range []struct {
-		manifest string
-		digest   string
-		fail     bool
+		config string
+		digest string
+		fail   bool
 	}{
-		// manifest pulled from docker hub blob store
+		// config pulled from docker hub blob store
 		//
-		// curl -L -H "Authorization: Bearer ..." -H \
-		// -H "Accept: application/vnd.docker.distribution.manifest.v2+json" \
-		// https://registry-1.docker.io/v2/library/docker/blobs/sha256:5359a4f250650c20227055957e353e8f8a74152f35fe36f00b6b1f9fc19c8861
+		// TOKEN=$(curl https://auth.docker.io/token\?service\=registry.docker.io\&scope\=repository:library/docker:pull  | jq -r .token)
+		// MANIFEST=$(curl -H "Authorization: Bearer ${TOKEN}" -H 'Accept: application/vnd.docker.distribution.manifest.v2+json' https://index.docker.io/v2/library/docker/manifests/1.12.1 | jq -r .config.digest)
+		// curl -LH "Authorization: Bearer ${TOKEN}" https://index.docker.io/v2/library/docker/blobs/${MANIFEST}
```

Comment:
When you dropped the prompt `$`, you went back to `MANIFEST` for the variable name.  I still think that this variable should be [`CONFIG` or `CONFIG_DIGEST`][1].

[1]: https://github.com/opencontainers/image-spec/pull/370#discussion_r83314195